gensrc/thrift/FrontendService.thrift

```diff
@@ -600,6 +601,7 @@ struct TLoadTxnCommitRequest {
     9: optional list<Types.TTabletCommitInfo> commitInfos
     10: optional i64 auth_code
     11: optional TTxnCommitAttachment txnCommitAttachment
+    12: required i64 thrift_rpc_timeout_ms
```

Comment:
new fields should be `optional`, for compatibility.
And it need to be checked `isset` when use it.